src/main/java/org/generationcp/middleware/api/germplasm/GermplasmServiceImpl.java

```diff
@@ -338,6 +342,7 @@ public long countGermplasmMatches(final GermplasmMatchRequestDto germplasmMatchR
 
 	@Override
 	public void deleteGermplasm(final List<Integer> gids) {
+		this.germplasmListManager.performGermplasmListEntriesDeletion(gids);
```

Comment:
Can we `moveperformGermplasmListEntriesDeletion` to `GermplasmListService`?
That means we will create own methods there:
- `Void deleteGermplasmListData`
- Create own `updateGermplasmListData` using simple DAO update
This will make it simpler to eventually phase out `GermplasmListManager`